src/app/theme/toolbar.css

```diff
@@ -135,6 +135,20 @@
 		& button[aria-label="Add a task list"] svg {
 			margin-left: -3px;
 		}
+
+		& .ck-insert-table-dropdown-grid-box {
+			margin: var(--ck-insert-table-dropdown-box-margin);
+			border: 1px solid var(--ck-color-base-border) !important;
+
+			& .ck-button.ck-on,
+			& .ck-button:not(.ck-disabled):hover {
+				border-color: var(--ck-color-focus-border);
+				background: var(--ck-color-focus-outer-shadow);
```

Comment:
These two lines are not applied actually, therefore when hovering over a grid box it does not look like focused/hovered but like a not hovered / not focused one. Check the 3x3 grid box below. Also, the selected boxes do not get a proper border color from this rule.

![](https://user-images.githubusercontent.com/95343030/196188187-9f71e90f-451a-4945-8cf2-005290e38a31.png)

Btw, it's due to a space between `&` and class name but in case of border-color - I think there will be something more to add.